include/Actuators/ActuatorSigmoidGauss3p.h

```diff
@@ -0,0 +1,137 @@
+#ifndef BIORBD_ACTUATORS_ACTUATOR_SIGMOID_GAUSS_3P_H
+#define BIORBD_ACTUATORS_ACTUATOR_SIGMOID_GAUSS_3P_H
+
+#include "biorbdConfig.h"
+#include "Actuators/Actuator.h"
+
+namespace biorbd {
+namespace rigidbody {
+class GeneralizedCoordinates;
+class GeneralizedVelocity;
+}
+
+namespace actuator {
+
+///
+/// \brief Class ActuatorSigmoidGauss3p is a joint actuator type which maximum
+/// is 3 parameter gaussian (Gauss3p)
+///
+class BIORBD_API ActuatorSigmoidGauss3p : public Actuator
+{
+public:
+    ///
+    /// \brief Construct Gauss3p actuator
+    ///
```

Comment:
Make sure you update these comments. They are used by doxygen to produce the documentation